content/posts/2019-12-12-download-hawk.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Introducing DownloadHawk"
+date: 2019-12-12
+authors:
+- degeri
+tags: [Decred,security,binary verification]
+banner_image: "/images/posts/download_hawk_1.png"
+---
+
+We recently observed the Monero hack in which an attacker compromised getmonero[.]org and replaced one of the binaries with a malicious coinstealer. 
```

Comment:
Why not use a real URL+Link here?

Reply:
Its common practice to not link to compromised sites.

Reply:
Link to an article about the hack?

Reply:
Done